common/helper.py

```diff
@@ -486,3 +509,21 @@ def update_json(check_type, success, output, json_path):
         with open(path, "w") as f:
             json.dump(new_data, f, indent=4, sort_keys=True)
     return True
+
+
+def call_subprocess(cmd_list, shell=True, env=None, cwd=None, check=True,
```

Comment:
I think that cmd can be str too

Reply:
subprocess.run works with list of arguments by default.
That's why I use list.